### PR TITLE
Added Twitter logo, and rebalanced their order

### PR DESCRIPTION
**Pull request by propensive**
Review by @heathermiller. Not sure if the Twitter and MS logos need to
be approved by the organizations before we use them, but let me know if
you'd like me to chase up on getting that approval.

### CODE REVIEW

**Comment by heathermiller** [comment]
Hey, thank you very much for proposing this! ...Though indeed you're right that the logos aren't up because we need written permission from both companies to include them. We just finished Twitter's membership paperwork this week, so the next step is to get that written permission to include the logo. Though Morgen Stanley decided to avoid the whole hassle with getting permission to post their logo, because it's so much legwork. We're allowed to mention them in the minutes as far as I understand, but we're just not allowed to post their logo due to all of the effort involved in getting approval.

So, long story short, we won't be able to post MS, and we have to wait a bit to post Twitter's. We can update this PR though to only include Twitter's and I can approve and merge as soon as I get written permission. Would that be OK?

**Comment by propensive** [comment]
Yeah, that's no problem! Not surprised about MS, but I didn't realise the Twitter logo was an issue, seeing as it's used without permission *everywhere*... though I guess never in a "corporate" context!

**Comment by SethTisue** [comment]
@heathermiller time to merge?

**Comment by heathermiller** [comment]
No, unfortunately. We don't have written permission to post either. We'll never get permission for MS, but Twitter I should start the process for :-/

**Comment by heathermiller** [comment]
So, closing not because I don't appreciate this PR but because I can't ever post MS's logo.

**Comment by SethTisue** [comment]
@heathermiller but Jon removed MS's logo in the most recent commit

**Comment by SethTisue** [comment]
darnit. I misread your comment. sorry
